src/app/components/note/note.component.html

```diff
@@ -4,26 +4,24 @@
   (delete)="onDeleteClick($event)"
   [id]="note.id"
 ></app-delete-note-popup>
-<div class="note" >
-  <div (click)="onClick()"
-  >
-  <h2 class="note-title">
-
-  
-    <div>
-      <mat-icon
-      [ngClass]="{ 'chevron-down': selectedNote === note.id }"
-      class="chevron"
-      >chevron_right</mat-icon
-    >
-    {{ note.title }}
-    </div>
-  <mat-icon class="delete-icon" (click)="openCloseDeletePopup()">delete</mat-icon>
-
-  </h2>
-  <h4 class="note-date" *ngIf="selectedNote === note.id">{{ note.date }}</h4>
-  <p class="note-context" *ngIf="selectedNote === note.id">
-    {{ note.context }}
-  </p>
+<div class="note">
+  <div (click)="onClick()">
+    <h2 class="note-title">
+      <div>
+        <mat-icon
+          [ngClass]="{ 'chevron-down': selectedNote === note.id }"
+          class="chevron"
+          >chevron_right</mat-icon
+        >
+        {{ note.title }}
+      </div>
+      <mat-icon class="delete-icon" (click)="openCloseDeletePopup()"
+        >delete</mat-icon
+      >
+    </h2>
+    <h4 class="note-date" *ngIf="selectedNote === note.id">{{ note.date }}</h4>
+    <p class="note-context" *ngIf="selectedNote === note.id">
+      {{ note.context }}
```

Comment:
Should we also display the **Subject**?

Reply:
Why? Isn't it already clear when we look at the bolded subject on the left side?

Reply:
Why? Isn't it already clear when we look at the bolded subject on the left side?

Reply:
Apologies, Tea.